examples/validation-app/src/controllers/ping.controller.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
```

Comment:
I know this is generated automatically, but if it's not needed for the example, can it be removed?

Reply:
do you mean to remove copyrights for all files in this example?

Reply:
Oh no sorry, I meant the ping controller.